<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KlantenScherm</class>
 <widget class="QDialog" name="KlantenScherm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/icons/icons/balance.png</normaloff>:/icons/icons/balance.png</iconset>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>931</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(169, 169, 169);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <widget class="QLCDNumber" name="lcdGewicht">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>361</width>
      <height>126</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbKg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>75</y>
      <width>61</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Kg</string>
    </property>
   </widget>
   <widget class="QFrame" name="frmStable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbStatus">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbMax">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Max: 15kg</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMin">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Min: 100 gr</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMax_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>105</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Schaal: 5gr</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbProduct">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Product</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPrijspereenheid">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>356</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Prijs per eenheid</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPrijs">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>9</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbEuro">
    <property name="geometry">
     <rect>
      <x>835</x>
      <y>35</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>â‚¬</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPromo">
    <property name="geometry">
     <rect>
      <x>835</x>
      <y>15</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>PROMO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbVerkoperNaam">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>150</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Verkoper:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>945</x>
     <y>10</y>
     <width>326</width>
     <height>1006</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbLogo">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>5</y>
      <width>251</width>
      <height>206</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resource.qrc">:/icons/icons/Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="lstwTicket">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>190</y>
      <width>316</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbTicketNr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ticketnr.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbGeholpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>840</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>U werd geholpen door</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbToaal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>775</y>
      <width>296</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>10</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Totaal:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>306</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>805</y>
      <width>306</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>855</y>
      <width>306</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>865</y>
      <width>291</width>
      <height>126</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Bedankt voor uw vertrouwen.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Kauwstraat 28 - 9550 Sint Lievens Esse&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Tel: 054 25 44 66&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;e-mail: info@deneeuwigenbeenhouwer.be&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;www.deneeuwigenbeenhouwer.be&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbPix_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>925</width>
     <height>820</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/icons/icons/Logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbPix_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>925</width>
     <height>820</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
